README.md

```diff
@@ -35,11 +39,11 @@ These reside in the [unit features directory](features/unit)
 | @unit.responses.genesis           | REST Client Unit Tests for GetGenesis endpoint        |
 | @unit.responses.messagepack       | REST Client MessagePack Unit Tests                    |
 | @unit.responses.messagepack.231   | REST Client MessagePack Unit Tests for Indexer 2.3.1+ |
+| @unit.sourcemap                    | Test the sourcemap endpoint.                        |
 | @unit.tealsign                    | Test TEAL signature utilities.                        |
 | @unit.transactions                | Transaction encoding tests.                           |
 | @unit.transactions.keyreg         | Keyreg encoding tests.                                |
 | @unit.transactions.payment        | Payment encoding tests.                               |
-| @unit.dryrun.trace.application    | DryrunResult formatting tests.                        |
 
 ### Integration Tests
```

Comment:
@tzaffi Leaving as a visual reminder:  I think the following README sections ought to be updated to reflect the PR.
* https://github.com/algorand/algorand-sdk-testing/#running-tests
* https://github.com/algorand/algorand-sdk-testing/#integration-test-environment
* https://github.com/algorand/algorand-sdk-testing/#start-the-test-environment

If it's possible to consolidate or slim down content, I'm all for it.  My primary concern is to minimize stale information.

Since the PR already covers a lot of ground, if you'd prefer to address the README in a follow up PR, I'm on-board.  I'll follow your lead.

Reply:
these have been updated